backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacServer.java

```diff
@@ -29,4 +33,9 @@
                              @HeaderParam(X_RH_IDENTITY_HEADER) String rhIdentity
 
     );
+
+    @GET
+    @Path("/groups/{groupID}/") // trailing slash is required by api
+    @Produces("application/json")
+    Uni<Response> getGroup(@PathParam("groupID") UUID groupId, @HeaderParam(X_RH_IDENTITY_HEADER) String rhIdentity);
```

Comment:
We don't need the reactiveness here:
```suggestion
    Response getGroup(@PathParam("groupID") UUID groupId, @HeaderParam(X_RH_IDENTITY_HEADER) String rhIdentity);
```